docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.13-slim-bullseye
+FROM python:3.8.16-slim-bullseye
```

Comment:
[Looks like](https://peps.python.org/pep-0569/) this is indeed the latest patch release. 👍

Reply:
Does this match what we run in sentry.io production?